client/src/pages/Profile.js

```diff
@@ -5,111 +5,144 @@ import Typography from '@material-ui/core/Typography';
-        <div className={classes.profile_info}>
-          <div className={classes.profile_bgimg}></div>
-          <div>
-            <Avatar className={classes.avatar}/>
-          </div>
-        </div>
-        <div>
-          <Button className={classes.itemButton} fullWidth>Send Request</Button>
-        </div>
-      </div>
-      <div className={classes.dishes}>
-        <Paper className={classes.paper}>
-          <Grid container wrap="nowrap" spacing={2}>
-            <Grid item>
-              <Avatar>W</Avatar>
-            </Grid>
-            <Grid item xs zeroMinWidth>
-              <Typography noWrap>PLACEHOLDER</Typography>
-            </Grid>
-          </Grid>
-        </Paper>
-        <Paper className={classes.paper}>
-          <Grid container wrap="nowrap" spacing={2}>
-            <Grid item>
-              <Avatar>W</Avatar>
-            </Grid>
-            <Grid item xs>
-              <Typography noWrap>PLACEHOLDER</Typography>
-            </Grid>
-          </Grid>
-        </Paper>
-        <Paper className={classes.paper}>
-          <Grid container wrap="nowrap" spacing={2}>
-            <Grid item>
-              <Avatar>W</Avatar>
-            </Grid>
-            <Grid item xs>
-              <Typography>PLACEHOLDER</Typography>
-            </Grid>
-          </Grid>
+    <Grid container className={classes.root}>
+      <Paper className={classes.sidebar}>
+        <Grid item xs
+              className={classes.sidebarimg}
+              container
+              direction="column-reverse"
+              alignItems="center">
+          <Avatar src={chef} className={classes.large} />
+        </Grid>
+        <Grid item xs
+              className={classes.chefinfo_one}
+              container
+              direction="column"
+              alignItems="center">
+        <Typography variant='h6' align='center'>Atsushi Mikazuki</Typography>
+        <Typography variant='subtitle1' align='center'>Toronto, Canada</Typography>
+        </Grid>
+        <Divider variant="middle" />
+        <Grid item xs
+              className={classes.chefinfo_two}
+              container
+              direction="column"
+              alignItems="center">
+
+        <Typography variant='subtitle2' align='center' lineHeight={2} className={classes.chef_description}>Chef with 8 years of experience in Japanese cuisine. Recipient of awards and positive feedback.</Typography>
+        </Grid>
+        <Button variant="contained" className={classes.srButton} fullWidth>Send Request</Button>
+      </Paper>
+
+      <Grid item xs
+            className={classes.dishes_section}
+            container
+            direction="column"
+            alignItems="center">
+        <Typography variant='h6' align='center' className={classes.chefs_name}>Atsushi's Menu:</Typography>
+        <Paper className={classes.dishes}>
+          <List className={classes.dishes_list}>
+            {[0, 1, 2].map((sectionId) => (
```

Comment:
let's use a variable for this right now that's defined above the component, so that later when we make this dynamic it'll be easier